server/src/types/song.types.ts

```diff
@@ -0,0 +1,21 @@
+import { Types } from 'mongoose';
+import { MongoInjectedType } from './mongo.types';
+
+type SongType = {
```

Comment:
should be `SongSchema` as what is discussed in the group:

___Document: This name is commonly used in Mongoose to represent a document in a collection, and may be more appropriate if you plan to use this type to interact with a database.
___Schema: This name emphasizes that the type is defining the schema for a group, which may be more appropriate if you are using this type to define the structure of a document in a NoSQL database.

we dont put type in the name because it's redundant. when u hover over the variable, it will say in vscode that its a type